client/src/components/Login/Login.js

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components/macro'
+import { ReactComponent as Spotify } from '../../assets/spotify.svg'
+import { ReactComponent as Logo } from '../../assets/resonance.svg'
+
+export default function Login() {
+  return (
+    <LoginWrapper>
+      <LoginGrid>
+        <LogoWrapper>
+          <Logo />
+        </LogoWrapper>
+        <Content>
+          <a href="http://localhost:3001/login" data-testid="login-button">
+            <LoginButton>
+              <p>
+                Login with <SpotifyLogo />
+              </p>
+            </LoginButton>
+          </a>
```

Comment:
This renders 
```html
<a>
  <button>
    <p>
      Login with <svg .../>
    </p>
  </button>
</a>
```

I think you can reduce this to:
```html
<a>
  Login with <svg .../>
</a>
```

in React:

```suggestion
          <LoginLink href="http://localhost:3001/login" data-testid="login-button">
                Login with <SpotifyLogo />
          </LoginLink>
```

with a styled component
```js
const LoginLink = styled.a`
  ....
`;
```

OT :
If you use React Router (I didn't check), this should be a `<Link>`

Reply:
That's of course much better and clearer. Thank you!

I'm not sure whether the Router `Link` is right here. At this point I link to the outside and do not navigate within my app. What do you think? I would appreciate further feedback.

Reply:
So `http://localhost:3001/login` is not the React app.
You are completely right using `<a>` in this case.

You will face another issue because this will link to `localhost` in production too. 
Imagine deploying your app on `https://nina-peters.de/`. In this case, the link should target `https://nina-peters.de/login`

This can be solved by using a proxy in development:
https://create-react-app.dev/docs/proxying-api-requests-in-development/

Reply:
All right, thanks for all the input and the helpful resource! 
I'll adjust that in the next step when I revise my server structure and fix the deployment.